recipe/meta.yaml

```diff
@@ -5,8 +5,8 @@ package:
   version: {{ version }}
 
 source:
-  url: https://github.com/Unidata/netcdf4-python/archive/v{{ version }}rel.tar.gz
-  sha256: f6f2fc2dd80f72b5750290e8c09789bfa598cc4d1eed24c43009950624ead0b2
+  url: https://pypi.io/packages/source/n/netCDF4/netCDF4-{{version}}.tar.gz
```

Comment:
The main reason for using github instead of PyPI is b/c now upstream is waiting for the wheels to build before publish the source dist on PyPI, that delays the `conda-forge` package quite a bit sometimes.